coreos-base/hard-host-depends/hard-host-depends-0.0.1.ebuild

```diff
@@ -50,8 +50,13 @@ RDEPEND="${RDEPEND}
 	dev-libs/libyaml
 	dev-libs/nspr
 	dev-libs/protobuf
+	dev-python/flit_core
+	dev-python/gpep517
 	dev-python/mako
+	dev-python/poetry-core
 	dev-python/setuptools
+	dev-python/tomli
```

Comment:
I'll drop this line - it's a leftover. It's probably going to be pulled into SDK anyway, but there's no need for it to be an explicit dependency.